### PR TITLE
Feature/rs9

### PR DESCRIPTION
**Pull request by Vasylevych**
Add second slider

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,75 +1,412 @@
...
+button{cursor:pointer;}
+button::-moz-focus-inner{padding:0;border:0;}
+a,a:visited{text-decoration:none;}
+a:hover{text-decoration:none;}
+ul li{list-style:none;}
+img{vertical-align:top;}
+h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight:inherit;}
+
+@font-face {
+	font-family: 'AvenirLTStd';
```
Comment (holovw) [inline index.css]: Just name it Avenir

File: index.css [hunk 2]
```diff
@@ -1,75 +1,412 @@
...
+	src: url('fonts/AvenirLTStd-Black.eot');
+	src: local('вє'), url('fonts/AvenirLTStd-Black.woff') format('woff'), url('fonts/AvenirLTStd-Black.ttf') format('truetype'), url('fonts/AvenirLTStd-Black.svg') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'AvenirLTStd';
```
Comment (holovw) [inline index.css]: Just name it Avenir here too, it's easier too remember

File: index.css [hunk 3]
```diff
@@ -1,75 +1,412 @@
...
+a:hover{text-decoration:none;}
+ul li{list-style:none;}
+img{vertical-align:top;}
+h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight:inherit;}
+
+@font-face {
+	font-family: 'AvenirLTStd';
+	src: url('fonts/AvenirLTStd-Black.eot');
+	src: local('вє'), url('fonts/AvenirLTStd-Black.woff') format('woff'), url('fonts/AvenirLTStd-Black.ttf') format('truetype'), url('fonts/AvenirLTStd-Black.svg') format('svg');
+	font-weight: normal;
```
Comment (holovw) [inline index.css]: Here is it's normal, and below 500, find what's the best practices to import fonts.

File: index.css [hunk 4]
```diff
@@ -1,75 +1,412 @@
...
+ min-height: 100%;
+ display: flex;
+ flex-direction: column;
 }
 
-a:focus, a:active {
-  outline: none;
+.wrapper {
+  background-color: #FAF9F9;
+  font-family: AvenirLTStd;
```
Comment (holovw) [inline index.css]: And here then will be just Avenir

File: index.css [hunk 5]
```diff
@@ -1,75 +1,412 @@
...
 
-html, body {
-  height: 100%;
+.header-flex {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header-logo a{
```
Comment (holovw) [inline index.css]: where is a space a { ?

File: index.css [hunk 6]
```diff
@@ -1,75 +1,412 @@
...
+  font-size: 28px;
+  color: #000;
```
Comment (holovw) [inline index.css]: As we discussed before, you should use 6 character model

File: index.css [hunk 7]
```diff
@@ -1,75 +1,412 @@
...
+  margin: 0 0 0 10px;
+}
+
+.header-navigation ul {
+  display: flex;
+}
+
+.header-navigation li a {
+  margin: 0 20px 0 0;
+  color: #000;
```
Comment (holovw) [inline index.css]: 6 character model

File: index.css [hunk 8]
```diff
@@ -1,75 +1,412 @@
...
+}
+
+.header-info ul {
+  display: flex;
+}
+
+.header-info li a {
+  margin: 0 20px 0 0;
+  color: #000;
```
Comment (holovw) [inline index.css]: 6 character model

File: index.css [hunk 9]
```diff
@@ -1,75 +1,412 @@
...
+}
+
+.baner-polaroid-collaction-item {
+  font-size: 32px;
+  margin: 0 0 15px 0 ;
+}
+
+.baner-polaroid-collaction-text {
+    font-size: 24px;
+    line-height: 120%;
```
Comment (holovw) [inline index.css]: Line height in percents it's a bad approach, use pixels

File: index.css [hunk 10]
```diff
@@ -1,75 +1,412 @@
...
+    margin: 0 0 15px 0;
+    text-align: center;
+    color: #444141;
+}
+
+.baner-polaroid-collaction-link a {
+  font-size: 24px;
+  color: #000;
```
Comment (holovw) [inline index.css]: 6 characters model

File: index.css [hunk 11]
```diff
@@ -1,75 +1,412 @@
...
+.blocke-sale {
+  margin: 10px 0;
+  position: relative;
+}
+.sale-slider {
+  overflow: hidden;
+}
+
+.blocke-sale-slider {
+   position: relative;
```
Comment (holovw) [inline index.css]: redundant space here

Comment (holovw) [reply index.css]: _position

File: index.css [hunk 12]
```diff
@@ -1,75 +1,412 @@
...
+.sale-slider-text h2 {
+  font-size: 25px;
+  margin: 20px 0 10px 0;
+}
+
+.sale-slider-text p {
+  font-size: 18px;
+}
+
+
```
Comment (holovw) [inline index.css]: only one line margin

File: index.css [hunk 13]
```diff
@@ -1,75 +1,412 @@
...
+  display: flex;
+  flex-direction: column;
 }
 
-/* ------------------------------------------------------------------------------------ */
\ No newline at end of file
+.follow-us {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (holovw) [inline index.css]: you should add empty line in the end of files

File: index.html [hunk 14]
```diff
@@ -1,26 +1,26 @@
...
-            <header>
-                <div class="header-flex">
-                    <div class="header-logo"><a href="#">RandomStor</a></div>
-                    <div class="header-navigation">
-                        <nav>
-                            <ul>
+        <header>
+             <div class="container">
+               <div class="header-flex">
+                  <div class="header-logo"><a href="#">LACOSTE<img src="img/lacoste-crocodiles-logo.svg" height="50px" width="40px" alt="logo lacoste"></a></div>
```
Comment (holovw) [inline index.html]: define height and width in css please

File: index.html [hunk 15]
```diff
@@ -29,25 +29,279 @@
...
                     </div>
                 </div>
+            
+                
+                <div class="blocke-sale">
+                    <div class="container">
+                        <div class="sale-slider">
+                            <div class="sale-slider-track">
+                            
+
```
Comment (holovw) [inline index.html]: only one line margin

File: index.html [hunk 16]
```diff
@@ -29,25 +29,279 @@
...
-            <footer>
 
+
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                   
+
+
```
Comment (holovw) [inline index.html]: only one line margin

File: index.html [hunk 17]
```diff
@@ -29,25 +29,279 @@
...
             </footer>
-        </div>
     </div>
     
   
     
-    <script src="js/script.js"></script>
+    <script src="index.js"></script>
 </body>
 </html>
```
Comment (holovw) [inline index.html]: you should end file with empty line

File: index.html [hunk 18]
```diff
@@ -29,25 +29,279 @@
...
+                            <p>Our Politics</p>
+                            <a href="#">Stamap</a>
+                            <a href="#">Teams & Condtions</a>
+                            <a href="#">Privacy policy</a>  
+                        </div>
+                    </div>
+                </div>
             </footer>
-        </div>
     </div>
```
Comment (holovw) [inline index.html]: one line margin

File: index.css (hunk 1 above)
Comment (Vasylevych) [reply index.css]: Done

File: index.css (hunk 2 above)
Comment (Vasylevych) [reply index.css]: Done

File: index.css (hunk 3 above)
Comment (Vasylevych) [reply index.css]: Done

File: index.css (hunk 4 above)
Comment (Vasylevych) [reply index.css]: Done

File: index.css (hunk 6 above)
Comment (Vasylevych) [reply index.css]: Done

File: index.css (hunk 7 above)
Comment (Vasylevych) [reply index.css]: Done

**Review by holovw — APPROVED** [review]

**Pull request merged**